File: StuLab1/Akamine/rep1/code/1/Prime.c
#include <stdio.h>

int main(int argc, char **argv)
{
    int x = 1;

    for (int i = 3; i < 1000 * 50; ++i)
    {
        int dividable = 0;
        for (int j = 2; j < i; ++j)
        {
            if (i % j == 0)
            {
                dividable = 1;
                break;
            }
        }

        if (dividable == 0)
        {
            x++;
        }
    }

    printf("The number of primes = %d\n", x);

    return 0;
}

File: StuLab1/Akamine/rep1/code/1/Prime.py
x = 1
for i in range(3, 1000*50):
    dividable = False

    for j in range(2, i):
        if i % j == 0:
            dividable = True
            break

    if not dividable:
        x += 1

print("The number of primes =", x)
